boards/shields/semtech_sx126xmb2xxs/semtech_sx126xmb2xxs_common.dtsi

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2024 Semtech Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/lora_lbm/sx126x.h>
+
+/ {
+    chosen {
+        zephyr,lora-transceiver = &lora_semtech_sx126xmb2xxs;
+    };
+    aliases {
+        lora-transceiver = &lora_semtech_sx126xmb2xxs;
+    };
+};
+
+// FIXME: Is there a way to reassign the node label arduino_spi from spi2 to spi3 ?
```

Comment:
we can remove this fixme, we can reasign the arduino_spi in the Arduino shield

Reply:
Fixed in arduino header @RobertGalatNordic  please have a look on commit: boards: arduino shield board agnostic